src/remediations/fifi.js

```diff
@@ -123,32 +163,44 @@ function getName (executor) {
 }
 
 function getStatus (executor) {
-    if (!executor.id) {
+    if (!executor.id && executor.rhcStatus === 'disconnected') {
         return 'no_executor';
     }
 
-    if (!executor.source) {
+    if (!executor.source && executor.rhcStatus === 'disconnected') {
         return 'no_source';
     }
 
-    if (!executor.receptor) {
+    if (!executor.receptor && executor.rhcStatus === 'disconnected') {
         return 'no_receptor';
     }
 
-    if (executor.receptorStatus !== 'connected') {
+    if (executor.receptorStatus !== 'connected' && executor.rhcStatus === 'disconnected') {
         return 'disconnected';
     }
 
     return 'connected';
 }
 
+function getType (executor) {
+    if (executor.id && executor.rhcStatus === 'disconnected') {
```

Comment:
I think it would be more robust to explicitly define executor.type = satellite or rhc when creating an executor "bucket" rather than inferring it indirectly from connection status, id, etc.